kotlin/src/test/kotlin/com/linecorp/armeria/server/FlowAnnotatedServiceTest.kt

```diff
@@ -0,0 +1,278 @@
+/*
+ * Copyright 2021 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server
```

Comment:
Not related to this PR, could you move all tests classes in this test package under `com.linecorp.armeria.server.kotlin`?

Reply:
Moved all tests under `com.linecorp.armeria.server.kotlin` package!